sig-contributor-experience/contribex-review-guidelines-on-test-infra.md

```diff
@@ -0,0 +1,123 @@
+    5. [Changes to `Label_sync` Path](#changes-to-label_sync-path)
+3. [Out of Scope](#out-of-scope)
+
+---
+
+## When a Review from SIG Contributor Experience is Required
+
+A review from SIG Contributor Experience is necessary if any changes introduced by a pull request to the test-infra repo:
+
+- Introduce changes to how humans interact with Prow and its components (changes that alter contributor user experience)
+- Introduce changes to how bot interact with Prow? (TODO: verify!)
+- Introduce changes to how Prow interacts with GitHub, or what information it broadcasts back to GitHub (in form of comments, etc, for example)
+- Alter the contribution workflows for project contributors.
+
+Examples include – adding new labels like `kind/foo` enabled/required by default, or changes that modify the issue/PR triage workflow used by project maintainers.
+
+---
+
+## Recommended Areas for ContribEx Review
+
+### Changes to [`config/prow/plugins.yaml`](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/plugins.yaml)
+
+#### Plugins under ContribEx Review
+
+Following plugins fall under ContribEx review:
+
+- trigger
+- owners
+- approve
+- help
+- size
+- label
+- blockades
+- slack
+- welcome
+- require_matching_label
+
+#### Plugins Configured for Specific Orgs/Repos
+
+**Organizations:**
+- kubernetes
+- kubernetes-client
+- kubernetes-sigs
+- etcd-io
+
+**Repositories:**
+- kubernetes/kubernetes
+- kubernetes/community
+- kubernetes/org
+- kubernetes/publishing-bot
+- kubernetes-sigs/contributor-playground
+
+#### External Plugins Configured for Specific Orgs/Repos
+
+**Organizations:**
+- kubernetes
+- kubernetes-sigs
+- kubernetes-csi
+- kubernetes-client
+
+---
+
+### Changes to [`config/prow/config.yaml`](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/config.yaml)
+
+- [slack_reporter_configs](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/config.yaml#L215-L226)
+- [branch-protection](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/config.yaml#L228-L636)
+- [tide](https://github.com/kubernetes/test-infra/blob/master/config/prow/config.yaml#L638-L844)
+- [github_reporter](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/config.yaml#L850-L853)
+
+---
+
+### Changes to [`config/prow/OWNERS`](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/OWNERS#L1) file
+
+Any changes to the reviewers/approvers or filters listed in the `config/prow/OWNERS` file.
+
+---
+
+### Changes to Prow CI Jobs and Their OWNERS Files
+
+Changes to the definition of the following Prow CI jobs files as well as their respective OWNERS files, owned directly or indirectly by SIG ContribEx:
```

Comment:
Many of these changes are going to be technical in nature.

Instead, what I'm looking for is a document outlining what it is that makes a change something SIG Contribex should review, I.E. "changes that impact contributor workflows / contributor facing behaviors need contribex tech lead review" or something like that.

It is not realistic to unilaterally review all changes to all of these files, plenty of changes will just be e.g. updating to the latest builds and should not require bottlenecking on contribex TLs.

But right now if I tell people "you need contribex review for this contributor facing workflow change" there's no actual written reference for this, and I'm not sure how many people in the project are aware of any such policy.